lib/Dialect/Substrait/IR/Substrait.cpp

```diff
@@ -68,6 +68,23 @@ LogicalResult AdvancedExtensionAttr::verify(
   return success();
 }
 
+LogicalResult mlir::substrait::IntervalYearMonthAttr::verify(
+    llvm::function_ref<mlir::InFlightDiagnostic()> emitError, int32_t year,
+    int32_t month) {
+  if (year < -100000 || year > 100000)
+    return emitError() << "year must be in a range of [-10,000..10,000] years";
+  return success();
```

Comment:
Yes, nice! `month` should be checked the same way (and be in `[120,000..120,000]`).